code/datums/status_effects/buffs.dm

```diff
@@ -140,10 +140,29 @@
 	alert_type = null
 	on_remove_on_mob_delete = TRUE
 	var/alive = TRUE
+	var/hostile_env = TRUE
+	var/timer_id
+
+/datum/status_effect/cult_master/on_apply()
+	. = ..()
+	SSshuttle.registerHostileEnvironment(owner)
+	timer_id = addtimer(CALLBACK(src, .proc/unregister_hostile_environment), 20 MINUTES)
+
+/datum/status_effect/cult_master/proc/unregister_hostile_environment()
+	if(QDELETED(src) || QDELETED(owner))
+		return
+	SSshuttle.clearHostileEnvironment(owner)
+	priority_announce("Nanotrasen divine security forces have reinforced in the local sector; shuttle functionality has been restored and purified to prevent further delays.", "Nanotrasen Security Division", SSstation.announcer.get_rand_alert_sound())
```

Comment:
```suggestion
	priority_announce("Nanotrasen divine security forces have reinforced in the local sector. Shuttle functionality has been restored and purified to prevent further delays.", "Nanotrasen Security Division", SSstation.announcer.get_rand_alert_sound())
```